app/tests/factories.py

```diff
@@ -0,0 +1,76 @@
+import uuid
+
+import factory
+from factory.alchemy import SQLAlchemyModelFactory
+from factory.fuzzy import FuzzyText
+
+from app.main.db.models import Body, Consignment, File, FileMetadata, Series, db
+
+
+class BodyFactory(SQLAlchemyModelFactory):
```

Comment:
this file defines all of our factory boy SQLAlchemyModelFactories, each corresponding to our SQLAlchemy Model classes defining our db tables.